pom.xml

```diff
@@ -58,7 +58,7 @@
         <checkstyle.suppressions.location>checkstyle/suppressions.xml</checkstyle.suppressions.location>
         <io.confluent.rest-utils.version>8.0.0-0</io.confluent.rest-utils.version>
         <conscrypt.version>2.5.2</conscrypt.version>
-        <jetty.servlets.version>9.4.53.v20231009</jetty.servlets.version>
+        <jetty.servlets.version>9.4.54.v20240208</jetty.servlets.version>
```

Comment:
this is a bit weird why we need to pin jetty.servlets.version here, there is already a version from common https://github.com/confluentinc/common/blob/5eb23708ee9bd1e7c78204da53a8bd976b3b464e/pom.xml#L66 that we can use, I would suggest to remove the pin version here and use the one from common

Reply:
removed the pinned version